packages/truffle-deployer/test/deployer.js

```diff
@@ -256,7 +261,7 @@ describe("Deployer (sync)", function() {
 
   it("emits block events while waiting for a tx to mine", async function() {
     this.timeout(15000);
-    const startBlock = await web3.eth.getBlockNumber();
+    await web3.eth.getBlockNumber();
```

Comment:
We don't need this either, right?

Reply:
Doesn't look like it

Reply:
I'm going to go ahead and do the drive-by cleanup to not have to wait for another cycle of discussion. I'm usually not fond of this, but will make an exception.